packages/playground/testUtils.ts

```diff
@@ -16,7 +16,7 @@ export const isBuild = !!process.env.VITE_TEST_BUILD
 
 const testPath = expect.getState().testPath
 const testName = slash(testPath).match(/playground\/([\w-]+)\//)?.[1]
-export const testDir = path.resolve(__dirname, '../../temp', testName)
+export const testDir = path.resolve(__dirname, '../../packages/temp', testName)
```

Comment:
due to this, jest.config.ts should be updated to replace `'<rootDir>/temp'`  with `'<rootDir>/packages/temp'`

Reply:
a5cb9400